docs/eventing/samples/kafka/source/README.md

```diff
@@ -0,0 +1,199 @@
+# Apache Kafka - Source Example
+
+Tutorial on how to use the `KafkaSource` with a Knative Serving `Service`
+
+## Build and Deploy Steps
+
+### Prerequisites
+
+1. Install the `KafkaSource` component to your configured cluster:
+   ```
+   kubectl apply -f https://github.com/knative/eventing-contrib/releases/download/v0.8.2/kafka-importer.yaml
+   ```
+2. Check that the `kafka-controller-manager-0` pod is running.
+   ```
+   $ kubectl get pods -n knative-sources
+   NAME                         READY     STATUS    RESTARTS   AGE
+   kafka-controller-manager-0   1/1       Running   0          42m
+   ```
+3. Check the `kafka-controller-manager-0` pod logs.
+   ```
+   $ kubectl logs kafka-controller-manager-0 -n knative-sources
+   2019/03/19 22:25:54 Registering Components.
+   2019/03/19 22:25:54 Setting up Controller.
+   2019/03/19 22:25:54 Adding the Apache Kafka Source controller.
+   2019/03/19 22:25:54 Starting Apache Kafka controller.
+   ```
+
+#### Apache Kafka Topic (Optional)
+
+1. If using Strimzi, you can can set a topic modifying
+   `source/kafka-topic.yaml` with your desired:
+
+- Topic
+- Cluster Name
+- Partitions
+- Replicas
+
+  ```yaml
+  apiVersion: kafka.strimzi.io/v1alpha1
+  kind: KafkaTopic
+  metadata:
+    name: knative-demo-topic
+    namespace: kafka
+    labels:
+      strimzi.io/cluster: my-cluster
+  spec:
+    partitions: 3
+    replicas: 1
+    config:
+      retention.ms: 7200000
+      segment.bytes: 1073741824
+  ```
+
+2. Deploy the `KafkaTopic`
+
+   ```shell
+   $ kubectl apply -f kafka/source/samples/strimzi-topic.yaml
+   kafkatopic.kafka.strimzi.io/knative-demo-topic created
+   ```
+
+3. Ensure the `KafkaTopic` is running.
+
+   ```shell
+   $ kubectl -n kafka get kafkatopics.kafka.strimzi.io
+   NAME                 AGE
+   knative-demo-topic   16s
+   ```
+
+#### Event Display
+
+1. Build and deploy the Event Display Service.
+   ```
+   $ kubectl apply -f source/event-display.yaml
+   ...
+   service.serving.knative.dev/event-display created
+   ```
+2. Ensure that the Service pod is running. The pod name will be prefixed with
+   `event-display`.
+   ```
+   $ kubectl get pods
```

Comment:
I failed here with `No resources found`

Note: I skipped the optional Strimzi steps. If i run the following it says the service is not running:

```
kubectl get service.serving.knative.dev
NAME            URL                                        LATESTCREATED         LATESTREADY   READY   REASON
event-display   http://event-display.default.example.com   event-display-f8c8d                 False   RevisionMissing
```

Reply:
hrm, weird

Reply:
That may be problem with istio/serving setup?